== TEST (2019-04-16 18:25:42) 
-- CODE:
Qf=Q0*(1+j)**t
Q0=1,500
j=36t
t=36
print(round(Qf,2))
-- ERROR:
File "XXXX", line 3
    j=36t
        ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:25:52) 
-- CODE:
Qf=Q0*(1+j)**t
Q0=1,500
j=36t
print(round(Qf,2))
-- ERROR:
File "XXXX", line 3
    j=36t
        ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:26:58) 
-- CODE:
Q0=1,500
j=36t
Qf=Q0*(1+j)**j
print(round(Qf,2))
-- ERROR:
File "XXXX", line 2
    j=36t
        ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:28:40) 
-- CODE:
Qf=1,500*(1+j)**t
j=36
print(round(Qf,2))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    Qf=1,500*(1+j)**t
NameError: name 'j' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:29:56) 
-- CODE:
Qf=1,500*(1+3)**36
print(round(Qf,2))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    print(round(Qf,2))
TypeError: type tuple doesn't define __round__ method
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:30:00) 
-- CODE:
Qf=1,500*(1+3)**36
print(round(Qf,2))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    print(round(Qf,2))
TypeError: type tuple doesn't define __round__ method
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:30:02) 
-- CODE:
Qf=1,500*(1+3)**36
print(round(Qf,2))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    print(round(Qf,2))
TypeError: type tuple doesn't define __round__ method
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:30:02) 
-- CODE:
Qf=1,500*(1+3)**36
print(round(Qf,2))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    print(round(Qf,2))
TypeError: type tuple doesn't define __round__ method
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:30:02) 
-- CODE:
Qf=1,500*(1+3)**36
print(round(Qf,2))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    print(round(Qf,2))
TypeError: type tuple doesn't define __round__ method
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:30:03) 
-- CODE:
Qf=1,500*(1+3)**36
print(round(Qf,2))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    print(round(Qf,2))
TypeError: type tuple doesn't define __round__ method
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:49:23) 
-- CODE:
Qf=1,500*(1+3)**36
print(round(Qf,2))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 2, in <module>
    print(round(Qf,2))
TypeError: type tuple doesn't define __round__ method
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:50:41) 
-- CODE:
Qf=1,500*(1+j)**t
j=10%
t=36
print(round(Qf,2))
-- ERROR:
File "XXXX", line 2
    j=10%
        ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:52:25) 
-- CODE:
Qf=1,500*(1+j)**t
input(digite j ou t)
print(round(Qf,2))
-- ERROR:
File "XXXX", line 2
    input(digite j ou t)
                 ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:52:45) 
-- CODE:
Qf=1,500*(1+j)**t
input("digite j ou t")
print(round(Qf,2))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    Qf=1,500*(1+j)**t
NameError: name 'j' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-04-16 18:52:48) 
-- CODE:
Qf=1,500*(1+j)**t
input("digite j ou t")
print(round(Qf,2))
-- EXECUTION TIME:
0.113115
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1670/2971/main.py", line 1, in <module>
    Qf=1,500*(1+j)**t
NameError: name 'j' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-04-16 18:54:48) 
-- CODE:
Qf=1500*(1+j)**t
input(float(10**3))
print(round(Qf,2))
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    Qf=1500*(1+j)**t
NameError: name 'j' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
